app/src/main/java/com/kickstarter/viewmodels/CommentsViewHolderViewModel.kt

```diff
@@ -84,6 +84,9 @@ interface CommentsViewHolderViewModel {
         /** Emits the current [OptimizelyFeature.Key.COMMENT_ENABLE_THREADS] status to the CommentCard UI*/
         fun isCommentEnableThreads(): Observable<Boolean>
 
+        /** Emits the current [OptimizelyFeature.Key.COMMENT_ENABLE_THREADS] status to the CommentCard UI*/
```

Comment:
The documentation here is not quite correct, as `isCommentReply()` does not depend on any feature flag `OptimizelyFeature.Key.COMMENT_ENABLE_THREADS]`, please update it.

Reply:
done